test/extended/image_ecosystem/s2i_php.go

```diff
@@ -58,23 +60,41 @@ var _ = g.Describe("[image_ecosystem][php][Slow] hot deploy for openshift php im
 				err = e2e.WaitForEndpoint(oc.KubeFramework().ClientSet, oc.Namespace(), "cakephp-mysql-example")
 				o.Expect(err).NotTo(o.HaveOccurred())
 
-				assertPageCountIs := func(i int) {
+				assertPageCountRegexp := func(priorValue string) string {
 					_, err := exutil.WaitForPods(oc.KubeClient().Core().Pods(oc.Namespace()), dcLabel, exutil.CheckPodIsRunning, 1, 4*time.Minute)
 					o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
 
-					result, err := CheckPageContains(oc, "cakephp-mysql-example", "", pageCountFn(i))
+					result, val, err := CheckPageRegexp(oc, "cakephp-mysql-example", "", pageRegexpCount, 1)
 					o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
 					o.ExpectWithOffset(1, result).To(o.BeTrue())
+					if len(priorValue) > 0 {
```

Comment:
even if you have no prior value you'd still want to confirm the current value is greater than zero.

Reply:
the regexp takes care of that ... I borrowed if from the change @wozniakjan pointed me to